src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -411,6 +405,33 @@ private static Boolean parseBoolean(final String potentialArgumentValue)
         return null;
     }
 
+    private List<String> getBooleanTabCompletions(String potentialArgumentValue)
+    {
+        if (potentialArgumentValue.isEmpty()) {
+            return Arrays.asList(new String[] {"true", "false"});
+        }
+        if ("true".startsWith(potentialArgumentValue)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)